bangazonapi/views/profile.py

```diff
@@ -177,15 +177,15 @@ def cart(self, request):
             try:
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
-                line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
+                lineitems = OrderProduct.objects.filter(order=open_order)
+                lineitems = LineItemSerializer(
+                    lineitems, many=True, context={'request': request})
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={
                                                 'request': request}).data
-                cart["order"]["line_items"] = line_items.data
-                cart["order"]["size"] = len(line_items.data)
+                cart["order"]["lineitems"] = lineitems.data
```

Comment:
There's still `lineitems` and `line_items` returning the same thing. You'll want to remove one of those keys from the response. Either remove this line or check what is being returned in the `OrderSerializer`

Reply:
Removed line 187 and preserved the return in OrderSerializer, which itself handles generation of "lineitems" key-value pair.